packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-fundamentals.md

```diff
@@ -0,0 +1,100 @@
+---
+id: About
+section: accessibility
+---
+
+# Accessibility fundamentals 
+
+Accessibility is a foundation of PatternFly, so if you contribute to PatternFly or use it for your products you should first familiarize yourself with our accessibility guidelines and recommendations.
+
+No user should feel left behind when using your product. The goal of accessible design is to remove barriers and create inclusive product experiences that work for everyone, regardless of ability. This is best achieved when accessibility is considered early in the design and development process. 
```

Comment:
Not a blocker: what do you think about adding onto the last sentence, something like, "This is best achieved when accessibility is considered early in the design and development process, but it's never too late to start considering it."?